cmd/cstorvolumeclaim/cstorvolumeclaim.go

```diff
@@ -363,7 +363,7 @@ func createCVR(
 			WithOwnerRefernceNew(getCVROwnerReference(volume)).
 			WithFinalizers(getCVRFinalizer()).
 			WithTargetIP(service.Spec.ClusterIP).
-			WithCapacity(volume.Spec.Capacity).
+			WithCapacity(volume.Spec.Capacity.String()).
```

Comment:
Why are we using string format here. Can we not use the same datatype across Maya for capacity?

Reply:
In this PR I am changing type of `cstorvolume capacity`. Changing other than `cstorvolume` capacity is outof scope in this PR.

Reply:
If requirement comes we will do in upcoming PRs.